lib/Runtime/Language/DynamicProfileInfo.cpp

```diff
@@ -901,11 +902,13 @@ namespace Js
         {
             if (isModByPowerOf2)
             {
-                divideTypeInfo[profileId] = divideTypeInfo[profileId].Merge(ValueType::GetInt(true));
+                divideTypeInfo[profileId] = divideTypeInfo[profileId]
+                                           .Merge(ValueType::GetInt(true));
             }
             else
             {
-                divideTypeInfo[profileId] = divideTypeInfo[profileId].Merge(ValueType::Float);
+                divideTypeInfo[profileId] = divideTypeInfo[profileId]
+                                           .Merge(ValueType::Float);
             }
```

Comment:
nit: DynamicProfileInfo.cpp contains some formatting changes that are unnecessary (changes when you originally put them into a macro?).